docs/getting-started/system-requirements.rst

```diff
@@ -216,6 +219,78 @@ See `ScyllaDB on the New AWS EC2 I4i Instances: Twice the Throughput & Lower Lat
 learn more about using ScyllaDB with i4i instances.
 
 
+i4g instances
```

Comment:
No need to list ARM base instances here
Im4gn and Is4gn are **supported**, but the recommended instances are only i3, i3en and i4i

Reply:
Why shouldn't we list them?

They aren't named i4g though.

Reply:
There are no i4g instances, there are im4gn and is4gen (IIRC, please check)